### DISCUSSION TITLE
EEPROM support for PicoCNC

### DISCUSSION THREAD

**Discussion by Overlights** [phil-barrett/PicoCNC#11] [Q&A] [answered]
Hi Phil,
I will soon start assembling the PicoCNC, waiting to also receive the "Teensy-CNC" Shield and proceed in the same way.
A few months ago I read in your blog the post about the EEPROM memory.
This one:
https://www.grbl.org/single-post/eeprom-do-i-need-it
So I later purchased the F-RAM module you recommended (FM24CL16B-GTR) and naturally I would like to solder it onto both shields, along with the other parts.
But I noticed that in the "Plugins" section of the WebBuilder page, it is not possible to enable the EEPROM for the PicoCNC, nor the FRAM option.
Can you clarify this?

**Comment by phil-barrett** [comment]
This is a question for Terje.  I will ask on the thread in core discussions.

**Reply by Overlights** [reply]
Thanks Phil, I hadn't thought of that.
Do you want me to ask Terje directly, in the GRBLHAL/Core section?

**Comment by phil-barrett** [comment]
Already [posted it](https://github.com/grblHAL/core/discussions/203?sort=new). It's a rather long thread but kind of interesting read to see how the web builder has evolved.

**Reply by phil-barrett** [reply, marked as answer]
Terje just replied, he will add it.  give it a little time and try again.

**Reply by Overlights** [reply]
Thanks Phil!
And thanks also to Terje...!
The EEPROM and FRAM settings for PicoCNC, now works.
